.travis.yml

```diff
@@ -2,8 +2,7 @@ language: python
 python:
     - "3.6"
 install:
-    - pip install -r requirements/test.txt
-    - pip install -e .
+    - pip install -e .[test]
```

Comment:
Do the tests still pass without the `-e`? They should (and we should remove it).

Reply:
They didn't! But fixed that now :-)

Reply:
And this is why Travis shouldn't use `-e` :)